infrastructure/modules/xc3/.env-grafana.tpl

```diff
@@ -51,7 +51,7 @@ GF_AUTH_GENERIC_OAUTH_AUTH_URL=https://${user_pool_domain}.auth.${region}.amazon
 GF_AUTH_GENERIC_OAUTH_TOKEN_URL=https://${user_pool_domain}.auth.${region}.amazoncognito.com/oauth2/token
 GF_AUTH_GENERIC_OAUTH_API_URL=https://${user_pool_domain}.auth.${region}.amazoncognito.com/oauth2/userInfo
 
-GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_PATH=(\"cognito:groups\" | contains([*], 'Admin') && 'Admin' || 'Viewer')
+GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_PATH=("cognito:groups" | contains([*], 'grafana-admin') && 'Admin' || contains([*], 'grafana-viewer') && 'Viewer' )
```

Comment:
@kashan-01 Where we are creating these two cognito groups?

Reply:
We're creating just one (Admin) in cognito and this condition only check if the user is Admin or viewer. The previous condition generating problem is some aspects, but now its working very well.